app/config.py

```diff
@@ -1 +1,7 @@
-API_VERSION = '/api/v1'
\ No newline at end of file
+import os
+
+
+API_VERSION = "/api/v1"
+
+DEBUG = os.environ.get("FLASK_DEBUG", "False") == "True"
+CORS_ALLOWED_ORIGINS = "https://localhost:5000" if DEBUG else ""
```

Comment:
I think this will need to be :3000, the origin of the frontend rather than the backend?
confuses the hell out of me